VMEncryption/mdc-recommendation/helper-scripts/MdcHandler.py

```diff
@@ -76,17 +76,22 @@ def detect_encryption_status(imds_metadata):
     confidential_vm_status = VerifyVMSupport.is_confidential_vm(imds_metadata)
 
     if (confidential_vm_status is True):
-        is_vm_compliant = False
-        policy_reasons['code'] = "ConfidentialVMNotSupported"
+        is_vm_compliant = True
+        policy_reasons['code'] = "Compliant"
         policy_reasons['phrase'] = "Confidential VMs are not supported by this recommendation."

```

Comment:
Should this reason be empty?

Reply:
For the Linux GC package, we always display the reason for compliance or non-compliance since it's difficult and time consuming to debug the native binary. For example, for compliant VMs, we display why they are compliant (whether ADE or HBE is being used). Similarly, I thought it might be better to display the reason for being an unsupported scenario too. 

I have appended the reason for NotSupported scenarios in my latest commit to indicate that the VM is being marked as compliant since it is not supported.